helpers/api/bepro-initializer.ts

```diff
@@ -8,6 +8,7 @@ export default class Bepro {
   networkFactory: NetworkFactoryV2;
 
   async init(network = false, erc20 = false, factory = false) {
+    console.log(network, erc20)
```

Comment:
lingering console log?

Reply:
it was the only solution I could think of without wasting so much time dealing with the warning that is being called but is not being used.